virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -15,7 +15,7 @@ xref:../../installing/installing_bare_metal/preparing-to-install-on-bare-metal.a
 ====
 * {VirtProductName} only supports {op-system} worker nodes. RHEL 7 or RHEL 8 nodes are not supported.
 
-* Infrastructure solutions offered by bare metal cloud providers are not supported.
+* You can install {VirtProductName} on AWS bare metal instances (Technology Preview). Infrastructure solutions offered by other bare metal cloud providers are not supported.
```

Comment:
revise:
Bare metal instances offered by other cloud providers are not supported.

Reply:
Please link this to the appropriate spot in OCP 4.9 release notes
> You can install {VirtProductName} on AWS bare metal instances (Technology Preview).